_posts/2016-03-12-make-your-data-obey.markdown

```diff
@@ -0,0 +1,200 @@
+---
+layout: post
+title: "Make Your Data Obey"
+date: 2016-03-12 10:00:00
+categories: obey data validation modelling
+author: kent.safranski
+cover: /assets/images/covers/obey.jpg
+description: Data is what programming is all about. Applications would be nothing without that beautiful i/o of data moving between modules, services, and data storage. But data is nothing if it's not consistent.
+comments: true
+disqus: true
+---
+
+Data is what programming is all about. Applications would be nothing without that beautiful i/o of data moving between modules, services, and data storage. But data is nothing if it's not consistent.
+
+**[Obey](https://github.com/TechnologyAdvice/obey) was developed specifically to address, and provide solutions for, data modelling and validation for modern web applications developed in JavaScript.**
+
+## Diving In
+
+Let's start with a simple example; validating a user account. There are some core data points we want to ensure, using Obey's `model` method to create a model object is how we start:
```

Comment:
Instead of semi, trite but could help readers from getting hung up on grammatical distractions.